### ISSUE TITLE
How to implement Layer-wise learning rate decay on ResNet?

### ISSUE THREAD

**Issue by jingzhengli** [pengzhiliang/MAE-pytorch#95]
Hi, thanks for the nice code.
How to implement Layer-wise learning rate decay on ResNet instead of ViT?
